### PR TITLE
Feature/popular people

### PR DESCRIPTION
**Pull request by MarcinkPawel**
Hi all, Popular People component is ready to be checked.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
   "version": "0.1.0",
   "private": true,
+  "homepage": "https://marcinkpawel.github.io/movies-browser/",
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "react-dom": "^18.2.0",
-    "react-router-dom": "^6.10.0",
+    "gh-pages": "^5.0.0",
```
Comment (marta-zebrowska) [inline package.json:10]: Great you did it, finally we can deploy our app :)

File: src/components/PopularPeople/styled.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  max-width: 1368px;
+  background: ${({ theme }) => theme.color.mercury};
+  margin: 40px auto 0;
+  padding: 16px;
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
```
Comment (marta-zebrowska) [inline src/components/PopularPeople/styled.js:8]: I think that before @media we should have one ampty line

File: src/components/PopularPeople/styled.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  }
+`;
+
+export const Header = styled.h1`
+  font-weight: 600;
+  font-size: 36px;
+  color: ${({ theme }) => theme.color.black};
+  line-height: 1.2;
+  margin-bottom: 24px;
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
```
Comment (marta-zebrowska) [inline src/components/PopularPeople/styled.js:21]: the same story here ;)

**Review by marta-zebrowska — APPROVED** [review]
Great job Paweł :)

File: src/components/PopularPeople/styled.js (hunk 2 above)
Comment (MarcinkPawel) [reply src/components/PopularPeople/styled.js:9]: done

File: src/components/PopularPeople/styled.js (hunk 3 above)
Comment (MarcinkPawel) [reply src/components/PopularPeople/styled.js:23]: Done :)

**Pull request merged**